### PR TITLE
fix: Bug snapshot convert

### PR DESCRIPTION
**Pull request by KarpovIvan**
## Checklist

- [ ] for CI changes:
  - [ ] structural/foundational changes signed off by [CI DRI](https://github.com/cmur2)
  - [ ] [ci.yml](https://github.com/camunda/camunda/blob/main/.github/workflows/ci.yml) modifications comply with ["Unified CI" requirements](https://github.com/camunda/camunda/wiki/CI-&-Automation#workflow-inclusion-criteria)

## Related issues

closes #

### CODE REVIEW

File: optimize/backend/src/it/java/io/camunda/optimize/service/snapshot/BackupServiceIT.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+import io.camunda.optimize.AbstractCCSMIT;
+import io.camunda.optimize.dto.optimize.rest.BackupInfoDto;
+import io.camunda.optimize.service.BackupService;
+import io.camunda.optimize.service.db.es.schema.ElasticSearchSchemaManager;
+import io.camunda.optimize.service.db.schema.IndexMappingCreator;
+import io.camunda.optimize.service.util.configuration.db.DatabaseBackup;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+public class BackupServiceIT extends AbstractCCSMIT {
```
Comment (RomanJRW) [inline optimize/backend/src/it/java/io/camunda/optimize/service/snapshot/BackupServiceIT.java:23]: @KarpovIvan - I can't see this test actually running on the CI - am I missing something?

**Review by RomanJRW — CHANGES_REQUESTED** [review]
Changes LGTM, but requesting changes until we know that the test is running and passing

File: optimize/backend/src/it/java/io/camunda/optimize/service/snapshot/BackupServiceIT.java (hunk 1 above)
Comment (KarpovIvan) [reply optimize/backend/src/it/java/io/camunda/optimize/service/snapshot/BackupServiceIT.java:23]: image
https://github.com/camunda/camunda/actions/runs/10960818622/job/30436333429
Tests passed in previous commit.

Comment (RomanJRW) [reply optimize/backend/src/it/java/io/camunda/optimize/service/snapshot/BackupServiceIT.java:23]: maybe I am just blind, or maybe it's just Friday 🙈

**Review by RomanJRW — APPROVED** [review]

**Comment by RomanJRW** [comment]
Approved, but now the test actually fails. @KarpovIvan - can you check this before merging?

**Review by RomanJRW — APPROVED** [review]

**Pull request merged**